homeassistant/components/yeelight/__init__.py

```diff
@@ -71,6 +72,7 @@
     vol.Optional(CONF_MODE_MUSIC, default=False): cv.boolean,
     vol.Optional(CONF_SAVE_ON_CHANGE, default=False): cv.boolean,
     vol.Optional(CONF_MODEL): cv.string,
+    vol.Optional(CONF_POWER_MODE_ICON, default=False): cv.boolean,
```

Comment:
Let's not make it optional.

Reply:
Why not optional ? I can change it for sure, just to understand the reason for future.